core-ui/src/components/Extensibility/ExtensibilityCreate.js

```diff
@@ -90,8 +90,55 @@ export function ExtensibilityCreateCore({
     resource: api,
   });
 
+  const prepareRules = schemaRules => {
+    const PREDEFINED_PATHS = [
+      'metadata.name',
+      'metadata.labels',
+      'metadata.annotations',
+    ];
+
+    const defaultNameField = {
+      path: 'metadata.name',
+      simple: true,
+      widget: 'Name',
+      required: true,
+      inputInfo: t('common.tooltips.k8s-name-input'),
+      extraPaths: ['metadata.labels["app.kubernetes.io/name"]'],
+    };
+    const defaultLabelsField = {
+      path: 'metadata.labels',
+      widget: 'KeyValuePair',
+      defaultOpen: false,
+    };
+    const defaultAnnotationsField = {
+      path: 'metadata.annotations',
+      widget: 'KeyValuePair',
+      defaultOpen: false,
+    };
+
+    const nameField = {
+      ...defaultNameField,
+      ...schemaRules.find(r => r.path === 'metadata.name'),
+    };
+    const labelsField = {
+      ...defaultLabelsField,
+      ...schemaRules.find(r => r.path === 'metadata.labels'),
+    };
+    const annotationsField = {
+      ...defaultAnnotationsField,
+      ...schemaRules.find(r => r.path === 'metadata.annotations'),
+    };
+
+    return [
+      nameField,
+      labelsField,
+      annotationsField,
+      ...schemaRules.filter(r => !PREDEFINED_PATHS.includes(r.path)),
+    ];
+  };
+
```

Comment:
![image](https://user-images.githubusercontent.com/46759954/195015262-1a576d61-df67-4b0a-a03d-f5dada2f596d.png)
Maybe we can move it to a separate file because this file is so so so big.